Makefile

```diff
@@ -148,6 +148,15 @@ e2e-sgx:
 e2e-gpu:
 	@$(GO) test -v ./test/e2e/... -ginkgo.v -ginkgo.progress -ginkgo.focus "GPU" -delete-namespace-on-failure=false
 
+terrascan:
```

Comment:
Hello! When users run terrascan locally, then do they need to install terrascan by themselves?

Reply:
yes, they do need that. I can add its installation to the ```install-tools``` Makefile target if needed. However, we don't have all tools installed by that target, e.g. kubectl, venv etc.

Reply:
Yeah, that's true. Just I was curious about that matter.